extra_requirements/svm.json

```diff
@@ -0,0 +1,3 @@
+{
+  "svm": ["pandas>=0.22.2,<0.24.2","openml==0.10.2"]
```

Comment:
shouldn't this also require sklearn?

Reply:
yes, you are right! 
I removed it first, because openml requires scikit-learn>=0.18.1 already. But yeah: Explicit > Implicit.